openlibrary/plugins/openlibrary/js/dialog.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Wires up dialog close buttons
+ */
+export default function dialogs() {
```

Comment:
`initDialogs` is more consistent; but I see you do that in a future commit :+1:

Reply:
I can do that as part of this commit. I think this got lost in my rebasing.